src/test/java/StringUtilsTest.java

```diff
@@ -0,0 +1,33 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class StringUtilsTest {
+    StringUtils stringUtils;
+    @Before
+    public void setup(){
+        stringUtils=new StringUtils();
+    }
+
+    @Test
+    public void testReverse_shouldReturn_String(){
```

Comment:
you also need to mention given. means when should is expected. 

testReverse_shouldReturnString_givenExpectedInput